src/popup/views/UnlockAccount/index.tsx

```diff
@@ -1,55 +1,153 @@
-import React, { useState } from "react";
+import React from "react";
 import { get } from "lodash";
-import { history } from "popup/App";
 import { useDispatch, useSelector } from "react-redux";
-import { useLocation } from "react-router-dom";
+import { useLocation, Link } from "react-router-dom";
+import styled from "styled-components";
+import { Formik } from "formik";
+
 import { confirmPassword, authErrorSelector } from "popup/ducks/authServices";
+import { history } from "popup/App";
+
+import { POPUP_WIDTH, EMOJI } from "popup/constants";
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/styles";
+
+import {
+  FormSubmitButton,
+  FormRow,
+  FormErrorEl,
+  FormTextField,
+} from "popup/basics";
+
+import Form from "popup/components/Form";
+
+const UnlockAccountEl = styled.div`
+  width: 100%;
+  max-width: ${POPUP_WIDTH}px;
+  box-sizing: border-box;
+  padding: 2rem 2.5rem;
+`;
+const HeaderContainerEl = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 2.5rem 0.25rem;
+  line-height: 1;
+`;
+const HeaderEl = styled.h1`
+  display: inline-block;
+  color: ${COLOR_PALETTE.primary}};
+  font-weight: ${FONT_WEIGHT.light};
+  margin: 0;
+  margin-left: 1rem;
+`;
+const StyledLinkEl = styled(Link)`
+  color: ${COLOR_PALETTE.primary};
+`;
+const UnorderedListEl = styled.ul`
+  list-style-type: none;
+  font-size: 0.8rem;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0;
+  padding-top: 0.25rem;
+`;
+const CustomFormTextField = styled(FormTextField)`
+  padding-right: ${(props) => (props.hasError ? "6rem" : "2.2rem")};
```

Comment:
Added `padding-right` for an error text input so that the user won't be able to type over the two emojis in error